### PR TITLE
Fix style

### PR DESCRIPTION
**Pull request by SoroushF79**
Very minor style fixes

### CODE REVIEW

**Review by JimothyGreene — CHANGES_REQUESTED** [review]
PEP8 standards want 2 blank lines between each function in Python

File: Drivers/Src/CANbus.c.save [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#include "CANbus.h"
```
Comment (JimothyGreene) [inline Drivers/Src/CANbus.c.save:1]: I'm not really sure what this file is, but we shouldn't track this with git

File: BSP/Simulator/Src/BSP_CAN.c [hunk 2]
```diff
@@ -136,12 +136,14 @@ uint8_t BSP_CAN_Read(CAN_t bus, uint32_t* id, uint8_t* data) {
     for (uint8_t i = 0; i < NUM_CAN; i++) {
             if (bus == i) {
                 fprintf(fp, "\n");
-            } else {
+            } 
+            else {
```
Comment (JimothyGreene) [inline BSP/Simulator/Src/BSP_CAN.c:140]: I'd prefer if else statements to be written like:

```
if (condition) {
// body
} else {
// body
}

**Review by JimothyGreene — CHANGES_REQUESTED** [review]
Few minor changes

**Review by JimothyGreene — CHANGES_REQUESTED** [review]
Looks good. Just merge `master` into this and double-check the new `MotorController.py` file since now it's been refactored.

File: BSP/Simulator/Hardware/MotorController.py [hunk 3]
```diff
@@ -9,6 +9,59 @@
...
+
+velocity_increase = 0.5 #how much CURRENT_VELOCITY is increased by in update_velocity()
+
+CURRENT_SETPOINT = 0.0 #set by user via MOTOR_DRIVE commands
+
+MAX_CURRENT = 1.0 #max available current, this is a percent of the Absolute bus current
+
+ABSOLUTE_CURRENT = 5.0  #physical limitation
+
+
```
Comment (JimothyGreene) [inline BSP/Simulator/Hardware/MotorController.py:64]: Remove the parts of this that are encapsulated in the CarState class

**Review by JimothyGreene — CHANGES_REQUESTED** [review]

**Review by ErickCortez98 — DISMISSED** [review]

**Review by JimothyGreene — DISMISSED** [review]

**Pull request merged**